scripts/restructure-ideas-image-urls.js

```diff
@@ -0,0 +1,38 @@
+const db = require('../src/db');
+
+async function updateImageUrls() {
+
+    try {
+
+        const ideas = await db.Idea.findAll();
+
+        console.log(`Going through all ideas to look for image-arrays including the URLs with the base ${process.env.IMAGE_URL_OLD_BASE_PATH}, to replace them with the URL base ${process.env.IMAGE_URL_NEW_BASE_PATH}`)
```

Comment:
Do we need this logging in production? I think we can remove it? It does not tell us anything new right that we wouldn't know without the logging. If it just communicates where we are in the process I would shorten it and reflect the end log.

Image URL update process started (old_base_var -> new_base_var).